share/config.py

```diff
@@ -57,7 +57,7 @@ def __init__(
         self.username = username
         self.password = password
         self.api_key = api_key
-        self.es_index_or_datastream_name = es_index_or_datastream_name
+        self.datastream = datastream
```

Comment:
Albeit this is called `datastream` can still receive an index name, is that correct? So the functionality is still there, we just renamed the setting field.

Reply:
yes: https://github.com/elastic/elastic-serverless-forwarder/pull/115/files#diff-e30b3f6b54e55786ed08b5bc55aba70c4014a75d87a61054560a945feee77223R293